src/core/co_obj.c

```diff
@@ -103,8 +111,15 @@ CO_ERR COObjWrBufStart(struct CO_OBJ_T *obj, struct CO_NODE_T *node, uint8_t *bu
 
     type = obj->Type;
     if (type->Write != NULL) {
+        /* mandatory: reset object */
         (void)COObjReset(obj, node, 0);
-        result = type->Write(obj, node, (void *)buffer, size);
+
+        /* optional: read from object */
```

Comment:
I think this comment should be "write from object".

Reply:
> I think this comment should be "write to object".

Thank you @jernejsk! I've just corrected that comment.